pkg/client/unversioned/request.go

```diff
@@ -238,6 +246,35 @@ func (r *Request) AbsPath(segments ...string) *Request {
 	} else {
 		r.path = path.Join(segments...)
 	}
+	r.setAbspath = true
+	return r
+}
+
+// Group
+func (r *Request) Group(group string) *Request {
+	r.group = group
+	r.setGroup = true
+	return r
+}
+
+// Version
+func (r *Request) Version(version string) *Request {
+	r.version = version
+	r.setVersion = true
+	return r
+}
+
+// GroupVersion reads in a group/version string and set the group and version
+func (r *Request) GroupVersion(groupVersion string) *Request {
+	groupAndVersion := strings.Split(groupVersion, "/")
+	if len(groupAndVersion) == 1 {
```

Comment:
I think @caesarxuchao has a utility function somewhere that does this-- should just call it if possible.